distribution/ddf-common/src/main/resources/bin/setenv-wrapper.conf

```diff
@@ -12,36 +12,34 @@ wrapper.commandfile = %KARAF_HOME%/bin/restart.jvm
 # Additional JVM parameters
 # note that n is the parameter number starting from 11 (1-10 is defined in ddf-wrapper.conf).
 # ### make sure there is no gap renumber remaining parameters if you add, remove or uncomment lines
-wrapper.java.additional.11 = -Dddf.home=%KARAF_HOME%
-wrapper.java.additional.12 = -Dddf.home.policy=%KARAF_HOME%/
-wrapper.java.additional.13 = -server
-wrapper.java.additional.14 = -Dfile.encoding=UTF8
-wrapper.java.additional.15 = -Dkaraf.instances=%KARAF_HOME%/instances
-wrapper.java.additional.16 = -Dkaraf.restart.jvm.supported=true
-wrapper.java.additional.17 = -Djava.io.tmpdir=%KARAF_HOME%/data/tmp
-wrapper.java.additional.18 = -Djava.util.logging.config.file=%KARAF_HOME%/etc/java.util.logging.properties
-wrapper.java.additional.19 = -XX:+UnlockDiagnosticVMOptions
-wrapper.java.additional.20 = -XX:+DisableAttachMechanism
-wrapper.java.additional.21 = -XX:+UnsyncloadClass
-wrapper.java.additional.22 = -Djava.awt.headless=true
+wrapper.java.additional.11 = -server
+wrapper.java.additional.12 = -Dfile.encoding=UTF8
+wrapper.java.additional.13 = -Dkaraf.instances=%KARAF_HOME%/instances
```

Comment:
❓ Will this be an issue with spaces?

Reply:
Nope, service wrapper can handle paths with spaces, and actually requires that they not be quoted

Reply:
Thanks for checking